packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -9,101 +9,120 @@ ouia: true
 import pfLogo from './pfLogo.svg';
 import pfLogoSmall from './pf-logo-small.svg';
 
-## Examples
 
-### Basic
+## Examples 
+
+### Basic cards
+
+Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`. You may omit these components as needed, but it is recommended to at least include a `<CardBody>` to provide details about the card item.
 
 ```ts file='./CardBasic.tsx'
 ```
 
-### With modifiers
+### Modifiers
+
+You can further modify the styling of the card's content by using the properties in the following example. In the example, you can enable each modifier by selecting its respective checkbox.
+
+Most modifiers can be used in combination with each other, with the exception of `isCompact` and `isLarge`, since they are contradictory.
 
 ```ts file='./CardWithModifiers.tsx'
 ```
 
-### With image and actions
+| Modifier | Description |
+| --- | --- |
+| isCompact | Modifies the card to include compact styling. Should not be used with isLarge. |
+| isFlat | Modifies the card to include flat styling. |
+| isRounded | Modifies the card to include rounded border styling. |
+| isLarge | Modifies the card to be large. Should not be used with isCompact. |
+| isFullHeight | Modifies the card so that it fills the total available height of its container. |
+| isPlain | Modifies the card to include plain styling, which removes the border and background. | 
+
+
+### Header images and actions
+
+You can include header images using `<Brand>` within `<CardHeaderMain>` and you can include header actions within `<CardActions>`.
```

Comment:
Since the example is only showing the use of the `Brand` component rather than it being necessary for this scenario, it might be better to say something along the lines of:

```suggestion
You can include `CardHeader` images within `<CardHeaderMain>`, and you can include header actions within `<CardActions>`. The following example includes an image using the [Brand](/components/brand) component.
```

(could also omit the "`CardHeader`" bit and just keep "header")

Reply:
We might also want to add here that the `hasNoOffset` prop can be used whenever the user wants to remove the negative margin applied to the `CardActions` so that the Actions can better align with a large card title or a tall image in the header, for example.

Reply:
FWIW, technically you can put whatever you want in `<CardHeaderMain>`. The reason it exists is because `<CardHeader>` is a flex layout, which means you can't just drop anything in `<CardHeader>` because they'll be flex items and flex items don't lay out like regular things - https://codepen.io/mcoker/pen/KKobLPz

So we created `<CardHeaderMain>` (not a flex layout) as an element supported in the header that you can just put regular stuff in. 

Also if it helps understand `hasNoOffset`, the reason we have that prop is because with a default card, using the default card title (font/font-size), and a plain/kebab action on the right, our original design requirement was to align the title text and kebab icon. We did that by adding a negative margin/offset to the actions element as @nicolethoen mentioned, which moves it so the title and kebab align. Here's a picture of that:

* dotted blue border around the header content area
* dotted red border around that kebab/action (showing how it's offset outside of the bounds of the header content area it's in
* solid red line showing the text/kebab icon alignment

<img width="890" alt="Screen Shot 2022-08-18 at 3 37 05 PM" src="https://user-images.githubusercontent.com/35148959/185490902-2673f335-7898-4a62-a219-750dcfe94190.png">

However, as @nicolethoen pointed out, that breaks when you are using something text or whatever in the header that is a different size than the default title, so we have that variation to remove the offset so a user can manage that themselves if they like.

Reply:
Thanks for this context! I just added some of this info in my last commit -- let me know if that sounds correct! To remove this negative margin, does hasNoOffset just need to be set to "true"? I couldn't quite tell from the example.